### PR TITLE
[vcpkg] Fix OSX CI by ensuring the downloads directory exists

### PR DESCRIPTION
**Pull request by BillyONeal**

### CODE REVIEW

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -621,7 +621,6 @@ ignition-msgs5:x64-linux=fail
 ignition-msgs5:arm64-windows=fail
 ignition-msgs5:arm-uwp=fail
 ignition-msgs5:x64-uwp=fail
-ignition-msgs5:x64-osx=fail
```
Comment (BillyONeal) [inline scripts/ci.baseline.txt:624]: This one seems to be broken and not broken constantly. Should we skip instead?

Comment (JackBoosY) [reply scripts/ci.baseline.txt:624]: @BillyONeal Yes.

**Review by JackBoosY — APPROVED** [review]

File: scripts/ci.baseline.txt (hunk 1 above)
Comment (BillyONeal) [reply scripts/ci.baseline.txt:624]: Done.

**Pull request merged**